src/components/nodes/intermediateTimerEvent/intermediateTimerEvent.vue

```diff
@@ -7,5 +7,26 @@ export default {
   mounted() {
     this.shape.attr('image/xlink:href', clockIcon);
   },
+  data() {
+    return {
+      dropdownData: [
+        {
+          label: 'Intermediate Timer Event',
+          nodeType: 'processmaker-modeler-intermediate-catch-timer-event',
+          dataTest: '',
```

Comment:
Tiny point but this property could be omitted completely. I don't mind it as it is though.